bin/citrea/src/main.rs

```diff
@@ -35,6 +35,10 @@ struct Args {
     #[arg(long)]
     testnet: bool,
 
+    /// Override network to run devnet directly.
+    #[arg(long)]
+    devnet: bool,
+
     /// Run the development chain
     #[arg(long, conflicts_with_all = ["testnet"])]
     dev: bool,
```

Comment:
should we get rid of this now? or rename it to something like local-testing etc

Reply:
I don't think it's neccessary. The development version vs devnet is known to blockchain savvy people.